src/styles.css

```
@@ -31,6 +31,33 @@
       transform: translateX(100%);
     }
   }
+
+  @keyframes gem-shine {
+    0% {
+      filter: brightness(1) drop-shadow(0 0 1px oklch(0.72 0.17 350 / 0.2));
```

Stylelint (17.2.0)

[error] Expected "0.2" to be "20%" (alpha-value-notation)

(alpha-value-notation)